Changelog.md

```diff
@@ -1,4 +1,4 @@
-### 0.7.1 (unreleased)
+#### 0.7.1 (unreleased)
```

Comment:
```suggestion
#8125 ## 0.7.1 (unreleased)
```

Reply:
No idea how that happened actually... I'll add back the standard-json related changelog entry anyways, right?

Reply:
ok!

Reply:
done